src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.html

```diff
@@ -111,7 +131,7 @@
                         </mat-chip-list>
                       </span>
                     </td>
-                    <td *ngFor="let score of key.result">{{score | number : '1.2-2'}}</td>
+                    <td *ngFor="let score of key.result">{{score | number : '1.' + leaderboardPrecisionValue + '-' + leaderboardPrecisionValue }}</td>
```

Comment:
May be you can have a method that will return the value `'1.' + leaderboardPrecisionValue + '-' + leaderboardPrecisionValue` , you can see how it is crossing std line length > 80. 

In general:

* Try to make html as simple as possible. 
* Method size as small as possible - otherwise it will grow over the time.